superset-embedded-sdk/src/index.ts

```diff
@@ -36,6 +36,8 @@ export type EmbedDashboardParams = {
   supersetDomain: string
   /** The html element within which to mount the iframe */
   mountPoint: HTMLElement
+  /** An id to identify the create iframe that will embed Superset */
+  iframeAttributes: Map<string, string> 
```

Comment:
This will require instantiating a `Map()` object. We should probably allow passing key-value pairs such as `{ class: "some-class" }`. Let's change the type to use `Record` which represents this. (This will also require changing the `forEach` construction below)

```suggestion
  iframeAttributes: Record<string, string> 
```

Reply:
From the first time I was trying to force only Iframe related attributes to be passed here, and I was not able to find a good way to do that, but with your suggestion I have found the Partial type and I could implement it. What do you think about my new implementation where only valid iframe attributes can be passed to it?

Reply:
Oh, cool idea